<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>7/4/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Sep 22 20:39:00 IST 2022</td>
</tr>
<tr><td>Total time:</td><td>6 seconds (6701 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestNG.SoftAssert_Testing.SoftAssrt_Test2()'><b>SoftAssrt_Test2</b><br>Test class: TestNG.SoftAssert_Testing</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [11] but found [10],
	expected [11] but found [100]
	at TestNG.SoftAssert_Testing.SoftAssrt_Test2(SoftAssert_Testing.java:41)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1836643674", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1836643674'><pre>java.lang.AssertionError: The following asserts failed:
	expected [11] but found [10],
	expected [11] but found [100]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at TestNG.SoftAssert_Testing.SoftAssrt_Test2(SoftAssert_Testing.java:41)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>TestNG.SoftAssert_Testing@4738a206</td></tr>
<tr>
<td title='TestNG.HardAssert.HardAssert_Test()'><b>HardAssert_Test</b><br>Test class: TestNG.HardAssert</td>
<td><div><pre>java.lang.AssertionError: expected [11] but found [10]
	at TestNG.HardAssert.HardAssert_Test(HardAssert.java:16)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1174092510", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1174092510'><pre>java.lang.AssertionError: expected [11] but found [10]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:907)
	at org.testng.Assert.assertEquals(Assert.java:917)
	at TestNG.HardAssert.HardAssert_Test(HardAssert.java:16)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>TestNG.HardAssert@47987356</td></tr>
<tr>
<td title='TestNG.Validations_Using_Assertions.boolenValidation()'><b>boolenValidation</b><br>Test class: TestNG.Validations_Using_Assertions</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [true] but found [false]
	at TestNG.Validations_Using_Assertions.boolenValidation(Validations_Using_Assertions.java:78)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1828928838", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1828928838'><pre>java.lang.AssertionError: The following asserts failed:
	expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at TestNG.Validations_Using_Assertions.boolenValidation(Validations_Using_Assertions.java:78)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>2</td>
<td>TestNG.Validations_Using_Assertions@7e7be63f</td></tr>
<tr>
<td title='TestNG.SoftAssert_Testing.SoftAssrt_Test()'><b>SoftAssrt_Test</b><br>Test class: TestNG.SoftAssert_Testing</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	expected [11] but found [10]
	at TestNG.SoftAssert_Testing.SoftAssrt_Test(SoftAssert_Testing.java:25)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace21982592", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace21982592'><pre>java.lang.AssertionError: The following asserts failed:
	expected [11] but found [10]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at TestNG.SoftAssert_Testing.SoftAssrt_Test(SoftAssert_Testing.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>TestNG.SoftAssert_Testing@4738a206</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestNG.TestNg_Annotation_Uses.B_UrlTest()'><b>B_UrlTest</b><br>Test class: TestNG.TestNg_Annotation_Uses</td>
<td></td>
<td>1</td>
<td>TestNG.TestNg_Annotation_Uses@66d3eec0</td></tr>
<tr>
<td title='TestNG.TestNg_Annotation_Order.A_lrtTest()'><b>A_lrtTest</b><br>Test class: TestNG.TestNg_Annotation_Order</td>
<td></td>
<td>0</td>
<td>TestNG.TestNg_Annotation_Order@7fee8714</td></tr>
<tr>
<td title='TestNG.TestNg_Annotation_Uses.A_lrtTest()'><b>A_lrtTest</b><br>Test class: TestNG.TestNg_Annotation_Uses</td>
<td></td>
<td>3</td>
<td>TestNG.TestNg_Annotation_Uses@66d3eec0</td></tr>
<tr>
<td title='TestNG.TestNg_Annotation_Order.B_UrlTest()'><b>B_UrlTest</b><br>Test class: TestNG.TestNg_Annotation_Order</td>
<td></td>
<td>0</td>
<td>TestNG.TestNg_Annotation_Order@7fee8714</td></tr>
<tr>
<td title='TestNG.Validations_Using_Assertions.TittleValidation()'><b>TittleValidation</b><br>Test class: TestNG.Validations_Using_Assertions</td>
<td></td>
<td>0</td>
<td>TestNG.Validations_Using_Assertions@7e7be63f</td></tr>
<tr>
<td title='TestNG.TestNg_Annotation_Uses.Test_Check()'><b>Test_Check</b><br>Test class: TestNG.TestNg_Annotation_Uses</td>
<td></td>
<td>0</td>
<td>TestNG.TestNg_Annotation_Uses@66d3eec0</td></tr>
<tr>
<td title='TestNG.TestNg_Annotation_Order.Test_Check()'><b>Test_Check</b><br>Test class: TestNG.TestNg_Annotation_Order</td>
<td></td>
<td>0</td>
<td>TestNG.TestNg_Annotation_Order@7fee8714</td></tr>
</table><p>
</body>
</html>